tests/spdx/parser/jsonlikedict/test_package_parser.py

```diff
@@ -11,12 +11,66 @@
 from spdx.model.checksum import Checksum, ChecksumAlgorithm
 from spdx.model.package import ExternalPackageRef, ExternalPackageRefCategory, PackagePurpose, PackageVerificationCode
 from spdx.model.spdx_no_assertion import SpdxNoAssertion
+from spdx.model.spdx_none import SpdxNone
 from spdx.parser.error import SPDXParsingError
 from spdx.parser.jsonlikedict.dict_parsing_functions import parse_list_of_elements
 from spdx.parser.jsonlikedict.package_parser import PackageParser
 
 
-def test_parse_package():
+@pytest.mark.parametrize(
+    "homepage, expected_homepage, download_location, expected_download_location, "
+    "copyright_text, expected_copyright_text, originator, expected_originator, supplier, expected_supplier",
+    [
+        (
+            "http://ftp.gnu.org/gnu/glibc",
+            "http://ftp.gnu.org/gnu/glibc",
+            "NOASSERTION",
+            SpdxNoAssertion(),
+            "NONE",
+            SpdxNone(),
+            "Organization: ExampleCodeInspect (contact@example.com)",
+            Actor(ActorType.ORGANIZATION, "ExampleCodeInspect", "contact@example.com"),
+            "NOASSERTION",
+            SpdxNoAssertion(),
+        ),
+        (
+            "NOASSERTION",
+            SpdxNoAssertion(),
+            "NONE",
+            SpdxNone(),
+            "Copyright 2008-2010 John Smith",
+            "Copyright 2008-2010 John Smith",
+            None,
+            None,
+            None,
+            None,
+        ),
+        (
+            "NONE",
+            SpdxNone(),
+            "http://ftp.gnu.org/gnu/glibc/glibc-ports-2.15.tar.gz",
+            "http://ftp.gnu.org/gnu/glibc/glibc-ports-2.15.tar.gz",
+            "NOASSERTION",
+            SpdxNoAssertion(),
+            "NOASSERTION",
+            SpdxNoAssertion(),
+            "Person: Jane Doe (jane.doe@example.com)",
+            Actor(ActorType.PERSON, "Jane Doe", "jane.doe@example.com"),
```

Comment:
This is kind of hard to comprehend due to the scrambling of normal, none and no assertion cases. I get that this might make the test cases more diverse, but I don't think it is really worth it compared to just having test cases where all values are normal values or None/NONE etc.

Reply:
Yes, my intention was to make the test cases more diverse. I think both options doesn't really make a difference and as they are also just test cases  I would stick to the current implementation and merge it like that.